### PR TITLE
UT: improve unit-test coverage for the titanium web-dialog module.

### PR DESCRIPTION
**Pull request by akansha-maheshwari-axway**
https://jira.appcelerator.org/browse/MOD-2601

### CODE REVIEW

**Comment by akansha-maheshwari-axway** [comment]
This PR is open on behalf of @aakash-jain-axway

File: test/unit/specs/module.spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+			});
+
+			it('returns a boolean', () => {
+				expect(webDialog.isOpen({})).toEqual(jasmine.any(Boolean));
+			});
+		});
+
+		if (IOS) {
+
+			webDialog = require('ti.webdialog');
```
Comment (fasihul-axway) [inline test/unit/specs/module.spec.js:49]: Why you are redefining `webDialog`?

Comment (aakash-jain-axway) [reply test/unit/specs/module.spec.js]: the change is done.

**Comment by fasihul-axway** [comment]
@aakash-jain-axway, as discussed you will update UT with auth-dialogue and web dialogue with some invalid values.

**Comment by aakash-jain-axway** [comment]
> @aakash-jain-axway, as discussed you will update UT with auth-dialogue and web dialogue for invalid values.

@fasihul-axway : Tried opening the web-dialog and also the authSession with invalid(also empty) urls but the error is being thrown by the internal api framework and not by module method itself, hence expect error from method unit-tests did not work out for this scenario.

File: test/unit/config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const host = Ti.Platform.osname === 'android' ? '10.0.2.2' : 'localhost';
+const port = '3210';
+module.exports.url = `http://${host}:${port}`;
```
Comment (janvennemann) [inline test/unit/config.js:3]: This file is not used anywhere so it can be removed.

File: test/unit/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const testsContext = require.context('./specs', true, /\.spec\.js$/);
+
+testsContext.keys().forEach(testsContext);
```
Comment (janvennemann) [inline test/unit/index.js:3]: This file is not used anywhere so it can be removed.

File: test/unit/config.js (hunk 2 above)
Comment (aakash-jain-axway) [reply test/unit/config.js]: All suggested changes are done and PR updated for same.

**Pull request merged**